packages/components/src/local/organisms/adjudication-messages-list/index.tsx

```diff
@@ -330,24 +333,28 @@ export const AdjudicationMessagesList: React.FC<PropTypes> = ({
       })
       setRows(dataTable)
 
-      if (!columns.length || !filter) {
-        console.time('LLOG_Collate Adjudicate Message Filters')
-        const umpireForce = forces.find((force: ForceData) => force.umpire)
-        // TODO: the column definitions should use the data collated in the column summary (below)
-        // provide more sophisticated column definition lookups
-        const summaryData = umpireForce && getColumnSummary(forces, umpireForce.uniqid, false, [])
-        const hideTurnColumn = turnFilter !== SHOW_ALL_TURNS
-        const columnsData: Column<AdjudicationRow>[] = !summaryData ? [] : [
-          { title: 'Reference', field: 'reference' },
-          { title: 'Turn', field: 'turn', type: 'numeric', hidden: hideTurnColumn }, //  },
-          { title: 'Complete', field: 'complete', render: renderBoolean },
-          { title: 'Important', field: 'important', lookup: { Y: 'Y', N: 'N' } },
-          { title: 'Owner', field: 'owner' },
-          { title: 'Order 1', field: 'order1', render: (row: AdjudicationRow) => renderOrderTitle(true, row) },
-          { title: 'Order 2', field: 'order2', render: (row: AdjudicationRow) => renderOrderTitle(false, row) },
-          { title: 'Activity', field: 'Reference' },
-          { title: 'Duration', field: 'period' }
-        ]
+      console.time('LLOG_Collate Adjudicate Message Filters')
+      const umpireForce = forces.find((force: ForceData) => force.umpire)
+      // TODO: the column definitions should use the data collated in the column summary (below)
+      // provide more sophisticated column definition lookups
+      const summaryData = umpireForce && getColumnSummary(forces, umpireForce.uniqid, false, [])
```

Comment:
@lebaphi  - I think that `getColumnSummary` could be expensive.  Can we only run it once we know we need to update the columns?
Hmm, no we can´t.  Never mind.

Reply:
Hello @philb89 - I've just been having a look at the above code.  We don't actually _use_ the `summaryData` above.  The columns aren't dependent on the data, and the only `lookup` we use is the `Y/N` for `important`.

So, the `needToUpdate` will never change.

I think that means that we only need to run this once, on application load.

Does that look right?

(I'm nervous of breaking something 🤣  )

Reply:
okay @IanMayo. if the table data does not change, we can ignore it 😄